modules/govuk_jenkins/templates/jobs/copy_data_from_production_to_aws.yaml.erb

```diff
@@ -24,15 +24,24 @@
         - build-discarder:
             days-to-keep: 30
             artifact-num-to-keep: 5
+        - slack:
+            notify-start: true
+            notify-success: true
+            notify-aborted: false
+            notify-notbuilt: false
+            notify-unstable: false
+            notify-failure: true
+            notify-backtonormal: false
+            notify-repeatedfailure: false
+            include-test-summary: false
+            room: <%= @slack_room %>
     scm:
       - env-sync-and-backup_Copy_Data_from_Production_to_AWS
     logrotate:
         numToKeep: 10
-    triggers:
-        - timed: 'H 11 * * *'
     builders:
         - shell: |
-            set +x
+            set -eu
```

Comment:
Why are these options changing? Was `set +x` included by accident?

Reply:
This was done to hide sensitive information from the console log.

Reply:
Ok, it would be good to split this and the similar change to the staging job out in to a separate commit, and include the above description in the commit message.

Reply:
Hi @issyl0 , I don't see the need to split this PR (given that it is already old and the purpose). If you still want this to be split and create a separate PR to remove `set -x`, please let me know.